### ISSUE TITLE
False negative when specializing type alias that includes TypeVarTuple and TypeVar

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#6904]
Pyright fails to generate an error for this case. This is part of the type system conformance test suite.

```python
from typing import TypeVar, TypeVarTuple

Ts = TypeVarTuple("Ts")
T1 = TypeVar("T1")
T2 = TypeVar("T2")

TA7 = tuple[*Ts, T1, T2]

v1: TA7[int]  # Type error: requires at least two type arguments
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.345, which I just published. It will also be included in a future release of pylance.
